t/sanity.t

```diff
@@ -565,3 +565,50 @@ hi
 --- response_body
 ok
 --- http09
+
+
+
+=== TEST 34: use the -a option to append the cookie field
+--- config
+    location /cookie {
+        more_set_headers -a 'Set-Cookie: name=lynch';
+        echo ok;
+    }
+--- request
+    GET /cookie
+--- response_headers
+Set-Cookie: name=lynch
+--- response_body
+ok
+
+
+
+=== TEST 35: the original Set-Cookie fields will not be overwritten, when using the -a option
+--- config
+    location /cookie {
+        more_set_headers 'Set-Cookie: name=lynch';
+        more_set_headers -a 'Set-Cookie: born=1981';
+        echo ok;
+    }
+--- request
+    GET /cookie
+--- raw_response_headers_like: Set-Cookie: name=lynch\r\n
```

Comment:
The cookie should contain the  'born=1981'

Reply:
got it